services/crates/crates-downloads.service.js

```diff
@@ -2,6 +2,7 @@
 
 const { downloadCount: downloadCountColor } = require('../color-formatters')
 const { metric } = require('../text-formatters')
+const { InvalidParameter, NotFound } = require('../../core/base-service/errors')
```

Comment:
We import and rexport a lot of `core` in the `services/index.js` so that they can be easily imported in all the various service modules, and to simplify refactorings when we shuffle/rename things in `core`
```js
const { InvalidParameter, NotFound } = require('..')
```

Reply:
When I saw that in your last suggestion I thought the `..` was just you leaving a placeholder 😆 Fixed!